pkg/controller/volume/expand/pvc_populator.go

```diff
@@ -99,16 +99,20 @@ func (populator *pvcPopulator) Sync() {
 		// Filter PVCs for which the corresponding volume plugins don't allow expansion.
 		pvcSize := pvc.Spec.Resources.Requests[v1.ResourceStorage]
 		pvcStatusSize := pvc.Status.Capacity[v1.ResourceStorage]
-		volumeSpec := volume.NewSpecFromPersistentVolume(pv, false)
-		volumePlugin, err := populator.volumePluginMgr.FindExpandablePluginBySpec(volumeSpec)
-		if (err != nil || volumePlugin == nil) && pvcStatusSize.Cmp(pvcSize) < 0 {
-			err = fmt.Errorf("didn't find a plugin capable of expanding the volume; " +
-				"waiting for an external controller to process this PVC")
-			populator.recorder.Event(pvc, v1.EventTypeNormal, events.ExternalExpanding,
-				fmt.Sprintf("Ignoring the PVC: %v.", err))
-			glog.V(3).Infof("Ignoring the PVC %q (uid: %q) : %v.",
-				util.GetPersistentVolumeClaimQualifiedName(pvc), pvc.UID, err)
-			continue
+		if volumeSpec := volume.NewSpecFromPersistentVolume(pv, false); volumeSpec.IsControllerExpandable() {
+			// A flex volume is expandable but since it resides on the kubelet not the controller
+			// and is dynamically probed plugin, FindExpandablePluginBySpec will never find it
+			// Therefore explicitly allow flex volumes to resize
+			volumePlugin, err := populator.volumePluginMgr.FindExpandablePluginBySpec(volumeSpec)
+			if (err != nil || volumePlugin == nil) && pvcStatusSize.Cmp(pvcSize) < 0 {
+				err = fmt.Errorf("didn't find a plugin capable of expanding the volume; " +
```

Comment:
same here

Reply:
@chakri-nelluri the same comment as above - there has been no change to this behavior.

I will update event type as mentioned above.

Reply:
ACK